### PR TITLE
Feat/dashboard chart tooltip

### PR DESCRIPTION
**Pull request by Tymek**
Initial version of new chart tooltip

image

### CODE REVIEW

File: frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx [hunk 1]
```diff
@@ -9,34 +9,88 @@ import {
...
+                            labels: {
+                                usePointStyle,
+                                pointStyle,
+                                textAlign,
+                                color,
+                            },
+                        } = chart?.legend?.options || {
+                            labels: {},
+                        };
+                        return (chart as any)
```
Comment (Tymek) [inline frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx:50]: I have some issues with types on Chart.js. Hopefully I'll be able to fix it in next iteration

File: frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx [hunk 2]
```diff
@@ -9,34 +9,88 @@ import {
...
-                    title: (tooltipItems: any) => {
-                        const item = tooltipItems?.[0];
-                        const date =
-                            item?.chart?.data?.labels?.[item.dataIndex];
-                        return date
-                            ? formatDateYMD(date, locationSettings.locale)
-                            : '';
-                    },
+                enabled: false,
+                external: (context: any) => {
```
Comment (kwasniew) [inline frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx:74]: is it gonna change to proper types?

File: frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx [hunk 3]
```diff
@@ -45,9 +99,16 @@ const createOptions = (theme: Theme, locationSettings: ILocationSettings) =>
             intersect: false,
             axis: 'x',
         },
+        elements: {
+            point: {
+                radius: 0,
+            },
+        },
+        // cubicInterpolationMode: 'monotone',
```
Comment (kwasniew) [inline frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx:107]: is it leftover or wip?

File: frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx [hunk 4]
```diff
@@ -9,34 +9,88 @@ import {
...
+) =>
     ({
         responsive: true,
         plugins: {
             legend: {
                 position: 'bottom',
+                labels: {
+                    boxWidth: 12,
+                    padding: 30,
+                    // usePointStyle: true,
```
Comment (kwasniew) [inline frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx:37]: is it leftover or wip?

**Review by kwasniew — APPROVED** [review]

File: frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx (hunk 3 above)
Comment (Tymek) [reply frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx:107]: WIP

**Pull request merged**